src/main/java/com/questhelper/QuestHelperConfig.java

```diff
@@ -311,7 +311,7 @@ default boolean showRuneliteObjects()
 		position = 0,
 		keyName = "highlightItemsBackground",
 		name = "Always highlight needed items",
-		description = "Highlight items you need for marked content type at all times",
+		description = "Highlight items you need for marked content types at all times",
```

Comment:
The wording for this full section is a bit confusing, imo, and I think that might be why people have enabled it & then wondered why items are highlighted when they don't have a quest active.

The 4 settings:
 - Always highlight needed items
 - Highlight active quest items
 - Highlight miniquest items
 - Highlight achievement diary items

all control the "background highlighter" only right

It might be a good idea to change the section name itself to something like "Unfinished quest hints"

Maybe something like this?

```diff
diff --git a/src/main/java/com/questhelper/QuestHelperConfig.java b/src/main/java/com/questhelper/QuestHelperConfig.java
index 48a62ae8..b5cc409e 100644
--- a/src/main/java/com/questhelper/QuestHelperConfig.java
+++ b/src/main/java/com/questhelper/QuestHelperConfig.java
@@ -302,16 +302,16 @@ public interface QuestHelperConfig extends Config
 
 	@ConfigSection(
 		position = 0,
-		name = "Item highlighting",
-		description = "Determines what items to highlight in the background"
+		name = "Unfinished quest highlights",
+		description = "Highlight things you will need for quests you have not yet completed"
 	)
 	String itemSection = "itemSection";
 
 	@ConfigItem(
 		position = 0,
 		keyName = "highlightItemsBackground",
-		name = "Always highlight needed items",
-		description = "Highlight items you need for marked content types at all times",
+		name = "Highlight missing items",
+		description = "Highlight items on the ground you might need for future quests or achievement diaries.<br>The settings below can be used to control if you want to show items from unfinished quests, miniquests, or diaries.",
 		section = itemSection
 	)
 	default boolean highlightItemsBackground()
@@ -322,8 +322,8 @@ public interface QuestHelperConfig extends Config
 	@ConfigItem(
 		position = 1,
 		keyName = "highlightNeededQuestItems",
-		name = "Highlight active quest items",
-		description = "Highlight every uncompleted quest's items you're missing on the floor",
+		name = "Include quest items",
+		description = "Highlight every uncompleted quest's items you're missing on the floor.<br>Requires the Highlight missing items to be enabled.",
 		section = itemSection
 	)
 	default boolean highlightNeededQuestItems()
@@ -334,8 +334,8 @@ public interface QuestHelperConfig extends Config
 	@ConfigItem(
 		position = 2,
 		keyName = "highlightNeededMiniquestItems",
-		name = "Highlight miniquest items",
-		description = "Highlight every uncompleted miniquest's items you're missing on the floor",
+		name = "Include miniquest items",
+		description = "Highlight every uncompleted miniquest's items you're missing on the floor.<br>Requires the Highlight missing items to be enabled.",
 		section = itemSection
 	)
 	default boolean highlightNeededMiniquestItems()
@@ -346,8 +346,8 @@ public interface QuestHelperConfig extends Config
 	@ConfigItem(
 		position = 3,
 		keyName = "highlightNeededAchievementDiaryItems",
-		name = "Highlight achievement diary items",
-		description = "Highlight every uncompleted achievement diary's items you're missing on the floor",
+		name = "Include achievement diary items",
+		description = "Highlight every uncompleted achievement diary's items you're missing on the floor.<br>Requires the Highlight missing items to be enabled.",
 		section = itemSection
 	)
 	default boolean highlightNeededAchievementDiaryItems()
```

![image](https://github.com/Zoinkwiz/quest-helper/assets/962989/2b326997-e262-4108-9da2-18e056942dbf)